Test/Core/Cloning/SpecificCloningTest.cs

```diff
@@ -23,7 +25,8 @@ namespace Duality.Tests.Cloning
 	[TestFixture]
 	public class SpecificCloningTest
 	{
-		[Test] public void CloneRegex()
+		[Test]
```

Comment:
Some auto-formatting went rogue in this file. Please revert the added newlines between the `[Test]` attributes and the test method. Regardless of whether or not this style is good, all the existing tests are written that way and we should remain consistent.

Reply:
Any way to prevent VS from doing this?